src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+@import "../utils/variables";
+
+.card {
+  display: flex;
+  width: 200px;
+  padding: 32px 16px 16px;
+  flex-direction: column;
+  align-items: center;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:nth-child(#{$cards-row}n) {
```

Comment:
The variable $cards-row is not defined in this file, make sure to define it or import it from another file.